docs/user-guide/configuring.md

```diff
@@ -1070,12 +1070,26 @@ Of particular note is that like `.gitignore` files, all paths used as patterns f
 
 Please see `.gitignore`'s specification for further examples of valid syntax.
 
-In addition to any patterns in a `.eslintignore` file, ESLint ignores files in `/**/node_modules/*` by default. It can still be added using `!`.
+In addition to any patterns in the `.eslintignore` file, ESLint always follows a couple implicit ignore rules even if the `--no-ignore` flag is passed. The implicit rules are as follows:
 
-For example, placing the following `.eslintignore` file in the current working directory will not ignore `node_modules/*` and ignore anything in the `build/` directory except `build/index.js`:
+* `node_modules/` as well as `bower_components/` directories are ignored.
```

Comment:
The default ignore patterns [have changed in v7.0.0](https://eslint.org/docs/user-guide/migrating-to-7.0.0#default-ignore-patterns-have-changed).

Reply:
Should be fixed in latest push.